Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1230,5 +1230,24 @@ protected virtual void Dispose(bool dispose)
 
             _disposed = true;
         }
+
+        public async ValueTask DisposeAsync()
+        {
+            await DisposeAsyncCore().ConfigureAwait(false);
+            Dispose(true);
```

Comment:
Why true here and false bellow?

Reply:
I *think* the proper way is passing false here and duplicating the code in the sync Dispose method in the async version.

Reply:
I think you're right, that fits a lot of what I was seeing looking at a lot of other examples